cmd/flow-dps-live/main.go

```diff
@@ -366,6 +381,10 @@ func run() int {
 		log.Info().Msg("Flow DPS Live Server stopped")
 	}()
 
+	// Here, we are waiting for a signal, or for one of the components to fail
+	// or finish. In both cases, we proceed to shut down everything, while also
+	// entering a go-routine that allows us to force shut down by sending
```

Comment:
```suggestion
	// entering a goroutine that allows us to force shut down by sending
```

Where does the spelling with a dash come from? Never seen it anywhere before